ansible_collections/arista/avd/plugins/action/yaml_templates_to_facts.py

```diff
@@ -37,6 +39,12 @@ def run(self, tmp=None, task_vars=None):
                     raise AnsibleActionFail("The argument 'templates' is not a list")
             else:
                 raise AnsibleActionFail("The argument 'templates' must be set")
+
+            output_filename = self._task.args.get("output_filename",False)
```

Comment:
Where is the file going to be saved? I think we need to specify that in the output_filename. When I tested this feature, the file(s) is in the playbook directory.

Reply:
I should change this variable name from `output_filename` to `dest` to match the regular ansible naming for file desitnations. It can be a relative or absolute path, since this is just parsed on to the `dest` argument of the `copy` module.